Jdownloader2.json

```
@@ -0,0 +1,56 @@
+{
+	"Jdownloader 2": {
+		"containers": {
+			"jdownloader2": {
+				"image": "jlesage/jdownloader-2",
+				"launch_order": 1,
+				"ports": {
+					"5800": {
+						"description": "WebUI Port",
+						"host_default": 5800,
+						"label": "Data Port",
+						"protocol": "tcp",
+						"ui": true
+					},
+					"3129": {
+						"description": "MyJdownloader Port",
+						"host_default": 3129,
+						"label": "MyJdownloader Port",
+						"protocol": "tcp"
+					}
+				},
+				"volumes": {
+					"/config": {
+						"description": "Choose a Share for configuration. Eg: create a Share called jdownloader-config for this purpose alone.",
+						"label": "Config Storage",
+						"min_size": 1073741824
+					},
+					"/output": {
+						"description": "Share where all the downloads will be",
+						"label": "Data Storage"
+					}
+				},
+				"environment": {
+					"PUID": {
+						"description": "UID to run Jdownloader ",
+						"label": "UID",
+						"index": 1
+					},
+					"PGID": {
+						"description": "Enter a valid GID to use along with the same UID. It(or the above UID) must have full permissions to all Shares mapped in the previous step.",
+						"label": "GID",
+						"index": 2
+					}
+				}
+			}
+		},
+              "description": "JDownloader 2 Rock-On using jlesage docker.<p>Based on the following docker image: <a href='https://github.com/jlesage/docker-jdownloader-2' target='_blank'>https://github.com/jlesage/docker-jdownloader-2</a>.",

```

Maybe we can use the description of the project as provided by the authors? The following below is the "official" description found on Jdownloader doc page:

```suggestion
              "description": "<p>JDownloader is a free, open-source download management tool with a huge community of developers that makes downloading as easy and fast as it should be. Users can start, stop or pause downloads, set bandwith limitations, auto-extract archives and much more. It's an easy-to-extend framework that can save hours of your valuable time every day!</p><p>Based on the following docker image: <a href='https://github.com/jlesage/docker-jdownloader-2' target='_blank'>https://github.com/jlesage/docker-jdownloader-2</a>.",
```